src/camvidlog2/ha_mqtt_client.py

```
@@ -0,0 +1,113 @@
+
+        self._received_message: str | None = None
+        self._subscribe_topic: str | None = None
+
+    def __enter__(self) -> Self:
+        self.client.loop_start()
+        self.client.connect(self.broker, self.port, 60)
+        # Create the event that will be set via callback
+        self._connect_event = threading.Event()
+        # Wait until connected
+        if not self._connect_event.wait(timeout=5):
+            raise TimeoutError("MQTT client could not connect to broker in time.")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)